### PR TITLE
[RM Ch10] Gaps sections updated

### PR DESCRIPTION
**Pull request by walterkozlowski**

### CODE REVIEW

**Comment by walterkozlowski** [comment]
As discussed at the RM meeting: I removed gaps related SFC and Acceleration, as these topic have been covered, and change the Multi-Cloud gap to focus on missing API standards.  Please review.

**Review by pgoyal01 — APPROVED** [review]

File: doc/ref_model/chapters/chapter10.md [hunk 1]
```diff
@@ -53,34 +51,17 @@ The built in mechanism in Kubernetes for scaling enterprise type of services and
...
-		
-There is also a need for a seamless integration with platforms, Kubernetes and OpenStack in particular. Kubernetes treats each device as unique HW and, hence, application developers are forced to specify vendor specific labels and utilise vendor specific libraries in their workload manifests as dependencies. OpenStack is attempting to address this gap with their [Cyborg](https://wiki.openstack.org/wiki/Cyborg) project.
-
-<a name="10.3.6"></a>
-### 10.3.6 Hybrid Multi-Cloud: Data Center to Edge
-In section "8.2.4 Multi-Cloud Interactions Model" there recognition that "To realise a federated cloud requires the definition and agreement on a set of APIs". Currently that are exists various:
-- Proprietry APIs, some of which have been adopted as default industry standards,
+<a name="10.3.64"></a>
+### 10.3.4 Hybrid Multi-Cloud: APIs
+Section "8.2.4 Multi-Cloud Interactions Model" defines several core roles within the Multi-Cloud Model and discusses stereo-typical interactions between them. However, the Model realises that a federated cloud requires the definition and agreement on a set of APIs. Currently there exist in the industry various:
```
Comment (ildikov) [inline doc/ref_model/chapters/chapter10.md:61]:
```suggestion
Section "8.2.4 Multi-Cloud Interactions Model" defines several core roles within the Multi-Cloud Model and discusses stereo-typical interactions between them. However, the Model realises that a federated cloud requires the definition and agreement on a set of APIs. The current fragmentation in the industry is caused by various factors:
```

**Comment by walterkozlowski** [comment]
@ildikov, thanks for the proposed improvement. I accepted it, can you please accept PR, thanks?

**Review by jbhartley — APPROVED** [review]
Reviewed for alignment with weekly meeting agreed updates.

**Review by ulikleber — APPROVED** [review]

**Pull request merged**